### ISSUE TITLE
3PC RichText TV textareas suddenly broken in 2.5.x

### ISSUE THREAD

**Issue by pixelchutes** [modxcms/revolution#13070]
### Summary
Since merging #13045, it seems any TV `textarea` rendered with 3PC (like Ace) are now broken.

The font + sizing is no longer the same, and the cursor positioning becomes invalid (off to the right), with new characters appearing at the wrong place during insertion.

### Step to reproduce
1. Ensure running MODX `2.5.x-dev` (62fe227197676a1bd158d327fb74252877f90341 or above)
2. Create TV textarea(s) for a given Template
3. Install Ace Extra, and activate for [use w/ TVs](https://github.com/danyaPostfactum/modx-ace/issues/6#issuecomment-43935637)
4. Edit Resource tied to above Template
5. Attempt editing Ace TV

### Observed behavior
- TVs rendered by Ace now show incorrect `font-family`.
- Cursor position is also invalid (appearing off to the right)
- Character insertion is incorrect (appearing off to the left)

#### Before:
<img width="900" alt="tv_before" src="https://cloud.githubusercontent.com/assets/352182/16881740/cfcbccba-4a78-11e6-962d-3e10123751bf.png">

#### After:
![tv_after](https://cloud.githubusercontent.com/assets/352182/16881699/abbd7fa8-4a78-11e6-93d4-4eb408b88fae.gif)

### Expected behavior
- Font should remain as before (`'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace;`).
- Cursor should show up where you would expect.
- New characters should follow the cursor.

Simply removing the recently added CSS resolves both the look and functionality:

```css
/* Offending CSS */
.modx-tv .x-form-textarea {
    font-family: inherit;
}
```

### Environment
MODX 2.5.x-dev

**Comment by Mark-H** [comment]
Perhaps the CSS definition in Ace could be more specific? Kinda an addon issue, tho it's annoying it's caused by a core change.

**Comment by pixelchutes** [comment]
Yeah, I debated where I should file this because of that. Unless there's a way to do it in core to adjust the specificity, potentially accounting for scenario like this? _(Long shot? 🏀)_

**Comment by pixelchutes** [comment]
MODX Extras aside, I personally preferred the `"Courier New", Courier, monospace` font of vanilla TV `textarea` as it was before. To each their own! Likely depends on how you're using it (such as for code, like JS/JSON/CSS, or simply text.) I could see how each could be better in different scenarios.

**Comment by pixelchutes** [comment]
Something like this appears to fix:

```css
.modx-tv .x-form-textarea:not(.ace_editor) {
    font-family: inherit;
}
```

**Comment by Mark-H** [comment]
Yuck.. :/

**Comment by pixelchutes** [comment]
See 🏀 above 😉

**Comment by pixelchutes** [comment]
@Mark-H OK, how about now? I rebased to be more generic. `:not(div)` works just the same, but maintains new core functionality from #13045. No more 3PC-specific mention(s). 👍

**Comment by Jako** [comment]
Merged into 2.5.1 (maybe 2.5.2 - since 2.5.1 is on release at this moment)
